client/src/plugins/version-info/ReleaseInfo.less

```diff
@@ -17,4 +18,20 @@
       margin-bottom: 0;
     }
   }
+
+  div {
+    font-weight: 500;
+    margin-bottom: 6px;
+  }
+
+  ul {
```

Comment:
If this is the new way to style lists, let us define it globally, in a shared space and re-use it across all components.

Reply:
I think the `What's new` message is not a globally used style. For such a (long) message, we might have more whitespace and other alignments as in other places, for instance, the file list or a list in a property description.

Reply:
My two cents is: Lists shall be designed in the same way everywhere. At least this shall be our default approach (start from being generic and specialize from there).

What's your opinion on this @andreasgeier?